cli/commons/src/Microsoft.Kiota.Cli.Commons/IO/IOutputFormatter.cs

```diff
@@ -10,12 +10,20 @@ public interface IOutputFormatter
     /// </summary>
     /// <param name="content">The string content to format and write out</param>
     /// <param name="options">The options to use when formatting output</param>
-    void WriteOutput(string content, OutputFormatterOptions options);
+    void WriteOutput(string content, IOutputFormatterOptions options);
 
     /// <summary>
     /// Format and write stream content
     /// </summary>
     /// <param name="content">The stream content to format and write out</param>
     /// <param name="options">The options to use when formatting output</param>
-    void WriteOutput(Stream content, OutputFormatterOptions options);
+    void WriteOutput(Stream content, IOutputFormatterOptions options);
```

Comment:
Any reason we can't replace this method with the overload below?

Reply:
See conversation above https://github.com/microsoft/kiota/pull/1326#discussion_r822631940